sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/SessionReceiverManager.cs

```diff
@@ -145,21 +145,24 @@ private async Task CreateReceiver(CancellationToken processorCancellationToken)
             }
         }
 
-        public override async Task CloseReceiverIfNeeded(
-            CancellationToken processorCancellationToken,
-            bool forceClose = false)
+        public override Task CloseReceiverIfNeeded(CancellationToken cancellationToken)
+        {
+            return ForceCloseReceiver(forceClose: true, cancellationToken);
```

Comment:
nit: can we still await this call within this method. We generally use this convention as it leads to more understandable stack traces in the event of exceptions.